kategory-docs/docs/docs/optics/fold/README.md

```diff
@@ -6,4 +6,58 @@ permalink: /docs/optics/fold/
 
 ## Fold
 
-TODO
\ No newline at end of file
+`Fold` is an optic that allows to focus into structure and get multiple results.
```

Comment:
`Fold` is an optic that allows you to focus into and structure to get multiple results.

Reply:
I just checked the other docs and I am using "focus" in a different way then I did for other docs. To be consistent with other docs it should be:

A `Fold` is an optic that can see into a structure and get 0 to N foci. Not sure if that clears up the bit about nested structures. Thoughts?

Reply:
Okay, consistency is better. Go for it!